main.js

```diff
@@ -10,33 +10,49 @@ wordCards.forEach((wordCard) => {
   Card(wordCard);
 });
 
-const main = document.querySelector('[data-js="main"]');
+const main = document.querySelector('[data-js="home"]');
 
-const riddles = [
-  { question: "Was ist der Unterschied zwischen einer Ente?", answer: "tja" },
-  { question: "Wie warm ist es über den Berg?", answer: "2Grad" },
-  { question: "Ja?", answer: "nein" },
-];
+let cards = [];
 
-riddles.forEach((riddle) => {
-  const questionCard = document.createElement("article");
-  questionCard.classList.add("card");
+const form = document.querySelector('[data-js="form"]');
+const cardsContainer = document.querySelector('[data-js="cards"]');
 
-  const question = document.createElement("h2");
-  question.innerText = riddle.question;
+form.addEventListener("submit", (event) => {
+  event.preventDefault();
 
-  const button = document.createElement("button");
-  button.classList.add("card__button");
-  button.innerText = "show answer";
-  button.setAttribute("data-js", "button");
+  const questionElement = form.elements.question;
+  const answerElement = form.elements.answer;
+  const tagsElement = form.elements.tags;
 
-  const answer = document.createElement("p");
-  answer.classList.add("card__answer", "hidden");
-  answer.innerText = riddle.answer;
-  answer.setAttribute("data-js", "answer");
+  const newCard = {
+    question: questionElement.value,
+    answer: answerElement.value,
+    tags: tagsElement.value
+      .split(",")
+      .map((tag) => tag.trim())
+      .filter((tag) => tag.length),
```

Comment:
Why do you apply `filter` 🤔 ? I think the filter method changes nothing or? Do you want to sort your tags? Maybe you want to work with the `sort ` method. [See MDN.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort#syntax)